src/main/webapp/app/entities/budget/budget.tsx

```diff
@@ -178,13 +178,25 @@ export class Budget extends React.Component<IBudgetProps, IEventBudgetState> {
           <ModalHeader toggle={this.toggleShowOptions} />
           <ModalBody>
             <h2 className="text-center">Options</h2>
-            <Button tag={Link} to={`${match.url}/${selectedEventBudgetId}/edit`} color="primary" className="d-block mx-auto my-3 w-75">
+            <Button
+              tag={Link}
+              to={`${match.url}/${selectedEventBudgetId}/edit`}
+              onClick={this.toggleShowOptions}
+              color="primary"
+              className="d-block mx-auto my-3 w-75"
+            >
               <FontAwesomeIcon icon="pencil-alt" />{' '}
               <span>
                 <Translate contentKey="entity.action.update">Update</Translate>
               </span>
             </Button>
-            <Button tag={Link} to={`${match.url}/${selectedEventBudgetId}/delete`} color="cancel" className="d-block mx-auto my-3 w-75">
+            <Button
```

Comment:
after click the confirm delete button in src/main/webapp/app/entities/budget/budget-delete-dialog.tsx, the componentDidMount() function in this file is not being executed, it causes the event budget and real amount not being updated

Reply:
resolved in 6887348aff486b1219e4b1902e403145b7bbf778